src/Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: fmt install lint test fmt-ci lint-ci install-dev
+
+fmt:
+	black . $(ARGS)
+
+install:
+	pip3 install --user -r requirements.txt
+
+install-dev:
+	pip3 install --user -r requirements_dev.txt
+
+lint:
+	flake8 .
+
+test:
```

Comment:
Could you add a CI file to run the tests on PR? Should be an example here: https://github.com/cds-snc/sre-bot/blob/main/.github/workflows/ci_code.yml

Reply:
Done. [CI file ](https://github.com/cds-snc/sentinel-forward-data-action/blob/sentinel_data_forwarder/.github/workflows/ci_code.yml) added and you can see the[ github action](https://github.com/cds-snc/sentinel-forward-data-action/actions/runs/3878260627) running and executing successfully.